db/dcrsqlite/apisource.go

```diff
@@ -1119,28 +1120,24 @@ func (db *wiredDB) GetExplorerBlock(hash string) *explorer.BlockInfo {
 		}
 		switch stake.DetermineTxType(msgTx) {
 		case stake.TxTypeSSGen:
-			stx := makeExplorerTxBasic(tx, msgTx, db.params)
+			stx := db.GetExplorerTx(tx.Txid)
```

Comment:
And this makes it way slower.  `makeExplorerTxBasic` involved no DB queries, now this involves tons of queries.

What all is needed?  All of TxInfo?

Reply:
Not all. This is what I need:
```
VinCount:  len(tx.Vin),
VoutCount: len(tx.Vout),
VoteValid: voteValid, // tx.Type really
TxID:      tx.TxID,
Total:     tx.Total,
Fees:      tx.Fee.ToCoin(),
Coinbase:  tx.Coinbase,
```

I can't get all of that from TxBasic, so I get full Tx, pass into `trimTxInfo()` in `explorer/websockethandlers.go` to get just what I need before passing into template.

Looking deeper at `GetExplorerTx(txid string)` in `db/dcrsqlite/apisource.go`, I think there may be a way to get the data I need without running all those many queries. I'll refactor (create another method  in `db/dcrsqlite/apisource.go` similar to `GetExplorerTx(txid string)` that returns what I need with fewer to 0 queries)